web/app/templates/tournament_new.html

```diff
@@ -0,0 +1,25 @@
+{% extends 'base.html' %}
+
+<h2>Please enter Tournament Info:</h2>
+
+{% block content %}
+<div class="">
+
+  <form class="" method="post" name='new'>
+    {{ form.hidden_tag() }}
+
+        <p>Event Name: {{ form.event_name() }}</p></br>
```

Comment:
Flask Forms comes with a nifty macro to generate these lines for you, so you don't have to redundantly write "Event Name:" into the template. See details here http://flask.pocoo.org/docs/0.11/patterns/wtforms/#forms-in-templates